### DISCUSSION TITLE
More help with 0.7, please: where did batch() go?

### DISCUSSION THREAD

**Discussion by nlfiedler** [leptos-rs/leptos#3330] [General]
Sorry, I'm probably going to have to keep asking questions because I keep running into things that I can't figure out from either the docs, examples, or the details in the 0.7 release notes.

What happened to the batch() function?

**Comment by benwis** [comment]
> Sorry, I'm probably going to have to keep asking questions because I keep running into things that I can't figure out from either the docs, examples, or the details in the 0.7 release notes.
> 
> 
> 
> What happened to the batch() function?

Eliminated, everything is batched now

**Reply by nlfiedler** [reply]
Oh, nice. Thank you.
